### DISCUSSION TITLE
听歌记录同步到听歌排行

### DISCUSSION THREAD

**Discussion by moonmemories** [SPlayer-Dev/SPlayer#204] [想法 Ideas]
刚使用了这款播放器两天，很是喜欢。但是发现听歌记录没有同步到官方听歌排行中。之前我也使用过YesPlayMusic，是可以同步听歌记录的。
然后我就去看了YesPlayMusic的源码，发现应用在听歌完成后会调用两次听歌打卡scrobble接口，开发者大佬看下是否可以参考下这个逻辑。

![微信图片_20240708110400](https://github.com/imsyy/SPlayer/assets/20583244/e4ae1f74-b9ae-408f-a2a2-36dd5f0ae24b)

**Comment by moonmemories** [comment]
@imsyy

**Reply by imsyy** [reply]
https://github.com/imsyy/SPlayer/blob/7ed33919ab450f3fdb1a801bf592653d920424b1/src/utils/Player.js#L267
目前已做处理，不知是否是在播放开始时就打卡的问题，新版会改为与 YesPlayMusic 一致的播放完后打卡
